server/lib/client-html.ts

```diff
@@ -408,7 +408,13 @@ class ClientHtml {
   }
 
   private static async addAsyncPluginCSS (htmlStringPage: string) {
-    const globalCSSContent = await readFile(PLUGIN_GLOBAL_CSS_PATH)
+    let globalCSSContent
```

Comment:
Prefer using `pathExists` from `fs-extra` to check if the file exists, and if not you can use your log message in `.info` mode

Reply:
Using try{}catch() can also detect permissions issues. I can add a `pathExists`, but I suggest to keep the try{}catch(). Would it be ok for you?

Reply:
From the node documentation:
https://nodejs.org/api/fs.html
«Using fs.exists() to check for the existence of a file before calling fs.open(), fs.readFile(), or fs.writeFile() is not recommended. Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file does not exist.»